nexus/src/app/sagas/project_create.rs

```diff
@@ -88,25 +88,42 @@ async fn spc_create_record(
         .map_err(ActionError::action_failed)
 }
 
+async fn spc_create_record_undo(
+    sagactx: NexusActionContext,
+) -> Result<(), anyhow::Error> {
+    let osagactx = sagactx.user_data();
+    let params = sagactx.saga_params::<Params>()?;
+    let opctx = OpContext::for_saga_action(&sagactx, &params.serialized_authn);
+
+    let (_authz_project, project) =
+        sagactx.lookup::<(authz::Project, db::model::Project)>("project")?;
```

Comment:
```suggestion
    let (.., project) =
        sagactx.lookup::<(authz::Project, db::model::Project)>("project")?;
```
Would this work?

Reply:
They're functionally the same; I was just being explicit about "what is the unused value" in this case since deserialization is particularly important on the `lookup` function, which is generic based on the type parameters.

Reply:
Also, to be pedantic in the context of `let` binding:

1. `let (.., foo) = ...` means "destructure and bind one or more values to the anonymous "...", but only bind the last value to `foo`"
2. `let (_, foo) = ...` means "destructure and bind exactly one value to `_`, and bind the second value to `foo`"
3. `let (_unused, foo) = ...` means "destructure and bind exactly one value to `_unused`, and bind the second value to `foo`"

Technically in cases (1) and (2), since the bind is anonymous, it also goes out of scope, and `drop` is called immediately on the unnamed values. In contrast, in (3), `drop` is only called on `_unused` when it actually goes out of scope.

However, I don't think `authz::Project` has a `drop` implementation, hence why this is the same.

So it's not *exactly* the same, but it's *basically* the same - I just used this for readability, since the types of values being pulled out of the `sagactx.lookup` are important to get right, and **caller-specified**.